packages/use-presence/src/index.tsx

```diff
@@ -12,20 +12,22 @@ type SeparateTransitionConfig = {
   exitTransitionDuration: number;
 };
 
+export type UsePresenceOptions = (
```

Comment:
Hmm, would it bother you much if we don't export this for now? I always find the naming of this quite hard (`UsePresenceOptions`, `PresenceOptions`, `TransitionConfig`, …). As long as it's private it's not that important. As you've mentioned there are ways in TypeScript how types can be generated based on a function signature.

Reply:
Alrighty, I walked back those updates 👍